core/src/forward_renderer.cpp

```diff
@@ -84,6 +85,31 @@ void ForwardRenderer::on_render() {
         m_lightData.push_spotlight_color_inner_rad(vec4(spotLight.get_color(), spotLight.get_inner_radius()));
     }
 
+    //=SKYBOX=================================
+
+    auto skyboxes = registry.view<InstanceMesh, SkyBox>();
+    for (auto& e : skyboxes) {
+        auto goOpt = scene.get_game_object_from_handle(e);
+        if (!goOpt) {
+            continue;
+        }
+
+        GameObject go = goOpt.value();
+        InstanceMesh& mesh = go.get_component<InstanceMesh>();
+        SkyBox& skyBox = go.get_component<SkyBox>();
+
+        bgfx::setVertexBuffer(0, mesh.get_vertex_buffer());
+
+        if (isValid(mesh.get_index_buffer())) {
+            bgfx::setIndexBuffer(mesh.get_index_buffer());
+        }
+
+        skyBox.set_uniforms();
+
+        bgfx::setState(BGFX_STATE_WRITE_RGB | BGFX_STATE_WRITE_A | BGFX_STATE_DEPTH_TEST_LEQUAL);
+        bgfx::submit(0, skyBox.get_program());
+    }
+
```

Comment:
Could we split this into functions?

Reply:
once IBL is ported restructuring of the forward render class will be made